packages/esm-service-queues-app/src/modals/edit-queue-entry.modal.tsx

```diff
@@ -38,6 +38,7 @@ const EditQueueEntryModal: React.FC<EditQueueEntryModalProps> = ({ queueEntry, c
             status: statuses.find((s) => s.uuid == formState.selectedStatus),
             priority: priorities.find((p) => p.uuid == formState.selectedPriority),
             priorityComment: formState.prioritycomment,
+            ...(formState.selectedProvider && { providerWaitingFor: { uuid: formState.selectedProvider } }),
```

Comment:
```suggestion
            providerWaitingFor: formState.selectedProvider ? { uuid: formState.selectedProvider } : null,
```

Reply:
Thanks for the reviews @denniskigen  let me look into them